src/site/stages/build/plugins/create-resources-and-support-section.js

```diff
@@ -291,10 +292,13 @@ function createArticleListingsPages(files) {
 }
 
 function createArticleResultSnippet(text) {
-  const sanitized = liquid.filters.strip_html(text);
-  const truncated = liquid.filters.truncate(sanitized, 200);
+  const sanitizedText = liquid.filters.strip_html(text);
+  const strippedNewlines = liquid.filters.strip_newlines(sanitizedText);
+  const decodedText = he.decode(strippedNewlines);
 
-  if (sanitized !== truncated) {
+  const truncated = liquid.filters.truncate(decodedText, 190);
+
+  if (decodedText !== truncated) {
```

Comment:
We could leverage the bug in `Cheerio.load` to decode these entities 😂 

I think I would have done something to ensure that if the truncation occurs in the middle of an entity, that we extend that truncation's slice index to the HTML entity's `;` so HTML entity doesn't get cut off. This makes sense to me but it's a little disappointing to see a new package required for this single line. 

How come `lodash/unescape` didn't do the job?

Reply:
Right? So `lodash/unescape` you can read [here](https://lodash.com/docs/4.17.15#unescape) only escapes the HTML entities &amp;, &lt;, &gt;, &quot;, and &#39;, but we actually use many more than that within those descriptions.

Reply:
Lodash then recommends using `he`, which is how I found that lib

Reply:
Ah okay! Thanks